package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ubiquibot/permit-generation",
-  "version": "1.3.1",
+  "version": "1.6.0",
```

Comment:
How is semver derived? Is it based on conventional commits prefixes?

Reply:
Correct, it reads the commit names and deduces the version.